iip-40.md

```diff
@@ -0,0 +1,61 @@
+```
+IIP: 40
+Title: Deprecate io1 Address Format
+Author: IoTeX Lab 
+Type: Standards Track
+Category Core
+Created: 2024-10-22
+``` 
+
+## Abstract
+
+This IIP proposes deprecating the native `io1...` address format in favor of the widely recognized Ethereum-compatible `0x...` format across the IoTeX ecosystem. After evaluating the pros and cons, we recommend a phased transition to enhance user experience, reduce confusion, and align with industry standards.
+
+## Motivation
+
+In 2020, IoTeX added support for the Ethereum-compatible `0x...` address format along with support for the Ethereum API, facilitating interactions with users, developers, and third-party tools familiar with Ethereum. Currently, IoTeX supports both `io1...` and `0x...` address formats. While existing community members are accustomed to this duality, new users often face confusion, leading to errors and support issues.
```

Comment:
please add a blurb to explain they are interchangble and share the same private key.